### PR TITLE
More efficient autoid rebase methord

### PR DESCRIPTION
**Pull request by shenli**
If we use dumped sql data to feed tidb, each insert statement will be 1MB text. It may contains thousand of rows in a single insert statement. Update autoid in the global kv for every row is too inefficient.
@zimulala @coocood PTAL

### CODE REVIEW

File: executor/executor_write.go [hunk 1]
```diff
@@ -682,6 +686,9 @@ func (e *InsertValues) getRowsSelect(cols []*table.Column) ([][]types.Datum, err
 		}
 		rows = append(rows, row)
 	}
+	if e.needRebase {
```
Comment (zimulala) [inline executor/executor_write.go]: `getRows` also calls `fillRowData`. So I think that it only puts this judgement into the `getRowsSelect` function isn't enough.

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**